app/controllers/katello/api/v2/activation_keys_controller.rb

```diff
@@ -215,7 +215,7 @@ def remove_subscriptions
     param :content_overrides, Array, :desc => N_("Array of Content override parameters") do
       param :content_label, String, :desc => N_("Label of the content"), :required => true
       param :value, String, :desc => N_("Override value. Provide a boolean value if name is 'enabled'"), :required => false
-      param :name, String, :desc => N_("Override key or name. Note if name is not provided the default name will be 'enabled'"), :required => false
+      param :override_name, String, :desc => N_("Override parameter key or name. Note if name is not provided the default name will be 'enabled'"), :required => false
```

Comment:
Hmm... warning! Incoming API change! Maybe we should deprecate the `:name` parameter instead of just removing it completely? I know Foreman does that sometimes. Do we?

Reply:
We should be able to look up a key by its name. Override name would be the name of the issue we want to override (i.e. enabled by default, priority, et al). Currently it is supplied by `name` but the issue has not made it to any katello release, so there would be literally no API change yet

Reply:
@cfouant I dont think we should be changing this server side. Cant the hammer pass it the right values?

Reply:
The lookup by name happens in `def index`, not `content_override`. This method should only take ID to identify the activation key, which is how you've implemented it. Without it making a release yet, I'm fine with this change.

Reply:
@parthaa - sure, I added this method to `before_action :find_activation_key` which didn't currently have it before. I think that should do the trick, testing now.